cloudwatch_dashboard_alb/main.tf

```diff
@@ -98,18 +121,21 @@ resource "aws_cloudwatch_dashboard" "alb" {
                 "view": "timeSeries",
                 "stacked": false,
                 "metrics": [
-                    [ "AWS/ApplicationELB", "HTTPCode_ELB_5XX_Count", "LoadBalancer", "${var.alb_arn_suffix}", { "period": 60, "stat": "Sum", "color": "#d62728" } ]
+                    [ "AWS/ApplicationELB", "HTTPCode_ELB_3XX_Count", "LoadBalancer", "${var.alb_arn_suffix}", { "period": 60, "stat": "Sum", "color": "#ff7f0e" } ],
+                    [ ".", "HTTPCode_ELB_4XX_Count", ".", ".", { "period": 60, "stat": "Sum", "color": "#1f77b4" } ],
+                    [ ".", "HTTPCode_ELB_5XX_Count", ".", ".", { "period": 60, "stat": "Sum", "color": "#d62728" } ]
+
                 ],
-                "region": "us-west-2",
-                "title": "5XX errors from ALB",
+                "region": "${var.region}",
+                "title": "Frontend errors from ALB",
```

Comment:
Do we consider the 3XX errors? We do a lot of redirects that are expected AFAIK.

Reply:
I guess it's a separate line so that doesn't matter. I misunderstood and thought we were calculating the sum of all 3xx, 4xx, and 5xxx errors.

Reply:
Yeah, that's why I hadn't included them in the original chart, because I wasn't sure if that was expected. But I ran a draft of this against production, and it looks like there are no 3XX responses or 4XX responses served by the ALB itself in the ordinary course of business. So I think given our current infra, having any 3XX/4XX/5XX responses from the ALB itself is an error condition. (None of these metrics include responses from the target group backends.)